consensus/ibft/sign.go

```diff
@@ -63,6 +72,7 @@ func signSealImpl(prv *ecdsa.PrivateKey, h *types.Header, committed bool) ([]byt
 	return seal, nil
 }
 
+// writeSeal creates and sets a signatures to Seal in IBFT Extra
```

Comment:
Typo here, did you mean `a signature`?

Reply:
Thank you! Fixed at 9192f447dc5a4e63cbceb1500a1690b6283d385d